server/protocols/jmap/src/test/java/org/apache/james/jmap/model/KeywordTest.java

```diff
@@ -0,0 +1,140 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
```

Comment:
According to JMAP spec:

```
 a keyword is a (case-sensitive) string of 1–255 characters
```

What happens with empty? With exactly 255 characters? With 256?